doc/source/user/problem_method.rst

```diff
@@ -635,5 +638,78 @@ parameters
+"maximum mass fraction" (specified by ``"accretion:max_mass_fraction"``).
+
+In ``"threshold"`` flavor accretion, the change in density of each cell zero if the current
+density is below the density threshold. If the current density is above the density threshold,
+the change in density is the current density minus the density threshold, or the maximum mass
+fraction times the current density, whichever is smaller.
+
+In ``"bondi_hoyle"`` flavor accretion, the density change in each cell is calculated according
+to the method described in Mark R. Krumholz et al 2004, ApJ, 611, 399. Furthermore, the
+density change is limited in the same way as in ``"threshold"`` accretion.
+
+In ``"flux"`` flavour accretion, the density change in each cell is calculated according to the
+method described in Andreas Bleuler & Romain Teyssier 2004, MNRAS, 445, 4015-4036.
+Furthermore, the density change is limited in the same way as in ``"threshold"`` accretion.
+
+This method can only be used if ``"merge_sinks"`` is also used, with ``"merge_sinks"`` preceding
+``"accretion"``. In addition, this method requires the use of three spatial dimensions.
+
+This method requires the following fields (in addition to the fields required by the hydro
+method): ``"density_source"``, ``"density_source_accumulate"``, ``"mom_dens_x_source"``,
+``"mom_dens_x_source_accumulate"``, ``"mom_dens_y_source"``, ``"mom_dens_y_source_accumulate"``,
+``"mom_dens_z_source"``, and ``mom_dens_z_source_accumulate"``. In addition, if sink particles
+have a ``"metal_fraction"`` attribute, there must be a ``"metal_density"`` field.
+
+This method also requires sink particles to have the following attributes: ``"mass"``, ``"x"``,
+``"y"``, ``"z"``, ``"vx"``, ``"vy"``, ``"vz"``, and ``"accretion_rate"``, which must all be
+of type ``"default"``.
+
+parameters
+----------
+
+.. list-table:: Method ``accretion`` parameters
+   :widths: 10 5 1 30
+   :header-rows: 1
+
+   * - Parameter
+     - Type
+     - Default
+     - Description
+   * - ``"accretion_radius_cells"``
+     - `float`
+     - `4.0`
+     - `The accretion radius (i.e., the radius of the spherical accretion zone)
+       in units of the minimum cell width (i.e., the minimum across all 3 dimensions),
+       at the highest refinement level. Its value must be less than one fewer than the minimum
+       ghost depth  for "flux" accretion, and less than the minimum ghost depth
+       for other flavors of accretion. The ghost depth is 4 (along all axes) by default.`
+   * - ``"flavor"``
+     - `string`
+     - ``""``
+     - `The flavor of accretion used, which can be either "threshold", "bondi_hoyle", or "flux". If this parameter is not set in the parameter file, or if some other string is provided, then the accretion method will be called but will do nothing.`
+   * - ``"density_threshold"``
+     - `float`
+     - `1.0e-6`
+     - `The value of the density threshold. The density in each cell in the accretion zone
```

Comment:
I did things like this to be consistent with the density threshold imposed by the `ppm` and `mhd_vlct` methods. It would be up to users to set this to be something physical (in code units).

One slight issue is that (for cosmological simulations), this threshold density will be a comoving rather than a proper density. However, I don't see a way around this if we have to ensure that this density threshold is not less than the density threshold imposed by the hydro method, which is comoving for cosmological simulations (as far as I know).

Reply:
It's very non-intuitive for the user though. I wonder how can we improve this? Could we instead make the density threshold an 'overdensity' - i.e. a multiple of the mean density? That would be good for cosmological runs. For non-cosmological runs the value can remain as user determined. Would that work?

Reply:
I agree that proper density in physical (e.g. cgs) units is a better way to specify the threshold than code units (comoving).

Reply:
What do you mean by 'density threshold imposed by the hydro method'?

Reply:
I should really have written "density floor" rather than "density threshold". Both the `ppm` and `mhd_vlct` methods have a `density_floor` parameter which sets a minimum value for the density.

Reply:
Yeah, the ``mhd_vlct`` enforces a density floor, based on the value of `Method:mhd_vlct:density_floor`.

The `ppm` method similarly also accepts `ppm:method:density_floor` as a parameter, but it ignores the value. I think it might enforce a floor of 1e-6 in some places.

As part of PR #174, I have implemented a Physics object that tries to track all of these floors in a centralized place. That PR is mostly done, but needs some polishing.  I'll try to do some polishing on that PR before next week so we can discuss and review it. 

If you want to get this PR merged sooner, I think you should probably duplicate the parameter for now.

Reply:
Ah okay makes sense. The thing is that I intend to allow the accretion method's "density threshold" to be independent of the hydro method's "density floor", with the caveat that the "density threshold" must be at least as large as the "density floor". So it would need to be a separate parameter in any case. However, having just one "density floor" parameter would make this check a bit simpler.

Reply:
Done, let me know what you think.

Reply:
Yes, I changed the code but I haven't updated the documentation accordingly, thanks for pointing this out.

Reply:
Ok, done now.